code/modules/cargo/packs/gun.dm

```diff
@@ -153,10 +160,31 @@
 	crate_name = "shotgun crate"
 	faction = FACTION_SRM
 
+/datum/supply_pack/gun/buckmaster
+	name = "Buckmaster Shotgun Crate"
+	desc = "For when you need to deal with 8 hooligans and can't be arsed to pump. Contains a semi-auto shotgun with an 8 round tube."
+	contains = list(/obj/item/storage/guncase/buckmaster)
+	cost = 3000
+	crate_name = "shotgun crate"
+
 /*
 		Rifles
 */
 
+/datum/supply_pack/gun/m12
+	name = "Sporter Rifle Crate"
+	desc = "Contains a recently manufactured Model 12 \"Sporter\", Serene Outdoor's premier small game rifle."
+	contains = list(/obj/item/storage/guncase/m12)
+	cost = 500
+	crate_name = "rifle crate"
+
+/datum/supply_pack/gun/m15
+	name = "Super Sporter Rifle Crate"
+	desc = "Contains a recently manufactured Model 15 \"Super Sporter\", Serene Outdoor's premier big game rifle."
```

Comment:
Same as with the Sporter. Calling a 5.56 a big game rifle might also be a little confusing, even if deer are technically big game- maybe just calling it a hunting rifle would work better. You probably shouldn't try hunting bear or moose with this thing!